### PR TITLE
SDIT-2093 Handle csip updated domain event

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/csip/CSIPService.kt [hunk 1]
```diff
@@ -63,6 +64,41 @@ class CSIPService(
...
+            nomisCSIPReportId = nomisCsip.nomisCSIPReportId,
+            mappingType = CSIPFullMappingDto.MappingType.DPS_CREATED,
+            attendeeMappings = listOf(),
+            factorMappings = listOf(),
+            interviewMappings = listOf(),
+            planMappings = listOf(),
+            reviewMappings = listOf(),
+          )
+        }
+      // TODO Check and update child mappings
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/csip/CSIPService.kt:94]: There is still additional work to do here - as new child mappings may have been created - will do in a separate PR

**Review by mikehalmamoj — APPROVED** [review]
